raddb/policy.d/canonicalization

```diff
@@ -111,3 +111,16 @@ rewrite_calling_station_id {
                 noop
         }
 }
+
+set_calling_station_id {
+        if (&User-Name && (&User-Name =~ /^${policy.mac-addr-regexp}$/i)) {
```

Comment:
I think one of Zayme's cousin is named Abdea Befafa
Which could translate to username abdeabefafa1

Jokes aside, I tend to agree with @fdurand on that point where it is extremely unlikely that will happen and since a few controllers tend to send «vidange requests» then we will fix 99.999999% of the cases and break the remaining. 

That seems like a good trade-off and likely the only one we can make as I don't see another way to address this.